src/screens/Login/Login.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { View, Text, Image, KeyboardAvoidingView } from "react-native";
+
+/* eslint-disable */
+import { Button, Input } from "components";
+import { validateEmail, validatePassword } from "utils/validators";
+import { styles } from "./styles";
+
+const logoImg = require("assets/logotype/logotype2x.png");
+const infoImg = require("assets/ic_info/ic_info2x.png");
+/* eslint-enable */
+
+const Login = props => {
+  const { email: emailFromProps, password: passwordFromProps } = props;
+  const [email, setEmail] = useState(emailFromProps);
+  const [emailError, setEmailError] = useState("");
+  const [password, setPassword] = useState(passwordFromProps);
+  const [passwordError, setPasswordError] = useState("");
+
+  const onLoginPress = () => {
+    props.onLogin({ email, password });
+  };
+  const onRegisterPress = () => {
+    props.onRegister();
+  };
+  const onPasswordForgot = () => {};
+  const onTextInputBlur = () => {
+    setEmailError(validateEmail(email));
+    setPasswordError(validatePassword(password));
+  };
+
+  return (
+    <KeyboardAvoidingView style={styles.container} behavior="position" enabled>
+      <View style={styles.headerContainer}>
+        <View style={styles.infoImgContainer}>
+          <Image style={styles.infoImg} source={infoImg} />
+        </View>
+        <View style={styles.header}>
+          <Image style={styles.logoImg} resizeMode="contain" source={logoImg} />
+          <Text style={styles.headerText}>
+            Центр Кольцевания Птиц при Акажемии наук Республики Беларусь
+          </Text>
+        </View>
+      </View>
+      <View>
+        <Input
+          value={email}
+          label="E-mail"
+          textContentType="emailAddress"
+          onChangeText={setEmail}
+          error={emailError}
+          onTextInputBlur={onTextInputBlur}
+        />
+        <Input
+          value={password}
+          label="Пароль"
+          textContentType="password"
+          onChangeText={setPassword}
+          error={passwordError}
+          onTextInputBlur={onTextInputBlur}
+          wrapperStyles={[styles.passwordInput]}
+        />
+        <Button caption="Войти" onPress={onLoginPress} appearance="Dark" />
```

Comment:
I've been thinking a while about current login validation flow and come to such conclusion:
Regexp validators for email and password field fit well to the registration form, whilst on login form there is enough just to check whether fields are not empty.
I would also like to notice that would be great if submitting form also entailed to field validation so that preventing user from submitting sign-in action with empty fields

Reply:
@AlexeyKaryka  There is validation for email on invision screens https://epam.invisionapp.com/d/main#/console/17142229/355351871/preview#project_console
Regarding the password I am agree, but Denis Kargin should clarify this.

Reply:
Oh, see it, thanks